sandbox-backend/src/test/java/com/exadel/sandbox/team5/service/impl/OrderServiceImplTest.java

```diff
@@ -0,0 +1,119 @@
+package com.exadel.sandbox.team5.service.impl;
+
+import com.exadel.sandbox.team5.dao.CompanyDAO;
+import com.exadel.sandbox.team5.dao.DiscountDAO;
+import com.exadel.sandbox.team5.dao.OrderDAO;
+import com.exadel.sandbox.team5.dto.DiscountDto;
+import com.exadel.sandbox.team5.entity.Employee;
+import com.exadel.sandbox.team5.mapper.MapperConverter;
+import com.exadel.sandbox.team5.service.DiscountService;
+import com.exadel.sandbox.team5.service.EmployeeService;
+import com.exadel.sandbox.team5.util.SecurityUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+class OrderServiceImplTest {
+
+    @Mock
+    EmployeeService employeeService;
+
+    @Mock
+    DiscountService discountService;
+
+    @Mock
+    MapperConverter mapperConverter;
+
+    @Mock
+    CompanyDAO companyDAO;
+
+    @Mock
+    DiscountDAO discountDAO;
+
+    @Mock
+    SecurityUtils securityUtils;
+
+    @Mock
+    OrderDAO entityDao;
+
+    @InjectMocks
+    OrderServiceImpl orderService;
+
+    @BeforeEach
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
As I remember it is not required if you use `@RunWith(MockitoJUnitRunner.class)`, could you please check it

Reply:
You are right, but @RunWith(MockitoJUnitRunner.class) doesn't initialyze mocks.